web/src/SidebarTriggerButton.test.tsx

```diff
@@ -44,23 +45,34 @@ describe("SidebarTriggerButton", () => {
       <SidebarTriggerButton
         isTiltfile={false}
         isSelected={true}
-        resourceName="doggos"
         triggerMode={TriggerMode.TriggerModeManualAfterInitial}
         hasBuilt={true}
         isBuilding={false}
         hasPendingChanges={false}
         isQueued={false}
+        onTrigger={() => triggerUpdate("doggos", "click")}
```

Comment:
Is this the place to test that using the keyboard shortcut triggers the correct resource?

Reply:
nope, that's more of the integration between the components and isn't really tested right now